saleor/graphql/order/types.py

```diff
@@ -37,6 +38,13 @@ class OrderEventOrderLineObject(graphene.ObjectType):
     item_name = graphene.String(description="The variant name.")
 
 
+class OrderSettings(CountableDjangoObjectType):
```

Comment:
I think it should be part of the settings module since it uses site settings model underneath.

Reply:
I meant the `saleor/graphql/shop/types.py` module. If we moved this type there, also the `OrderSettingsUpdate` could be moved to `shop/mutations`. Then we'll have all "settings" mutations in one place.